pallets/asset-manager/src/mock.rs

```diff
@@ -110,8 +110,8 @@ parameter_types! {
 pub type AssetId = u32;
 #[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, RuntimeDebug, TypeInfo)]
 pub enum MockAssetType {
-	MockAsset(AssetId),
 	Xcm(MultiLocation),
+	MockAsset(AssetId),
```

Comment:
Reason behind this change?

Reply:
I added a new `OldAssetType` in `migrations` and I wanted for both to encode the same way (first variant `Xcm`), so I can compare them in the tests.
```
pub(crate) enum OldAssetType {
	Xcm(xcm::v2::MultiLocation),
}
```